src/js/angular/core/templates/rdfResourceSearchTemplate.html

```diff
@@ -0,0 +1,25 @@
+<link href="css/rdf-resource-search.css?v=[AIV]{version}[/AIV]" rel="stylesheet"/>
+
+<div ng-if="getActiveRepository() && !isLoadingLocation() && hasActiveLocation()" ng-cloak>
+  <div class="card search_inputRDF" id="search_inputRDF">
+    <div class="card-block">
+      <search-resource-input class="search-b"
```

Comment:
@desislava-hristova-ontotext can we control how this `search-resource-input` directive handles search button click? Can we open results in separate tab? Maybe we should discuss if it's not better that way.

Reply:
It now opens in separate tab. I have added an input for the `search-resource-input` directive, to chose whether to open in new tab.